packages/runner/cypress/integration/retries.ui.spec.js

```diff
@@ -433,7 +433,7 @@ describe('runner/cypress retries.ui.spec', { viewportWidth: 600, viewportHeight:
     it('throws when set via this.retries in test', () => {
```

Comment:
We eval these suite functions, and since they're no longer compiled down to `'suite 1': function () {}`, e.g., evaling results in a syntax error if they're not explicitly function expressions.